cycler.py

```diff
@@ -335,7 +335,11 @@ def __repr__(self):
     def _repr_html_(self):
         # an table showing the value of each key through a full cycle
         output = "<table>"
-        sorted_keys = sorted(self.keys)
+        try:
+            sorted_keys = sorted(self.keys)
```

Comment:
What about:

```
sorted_keys = sorted(str(x) for x in self.keys)
```

that way they are sorted by their `repr` even if not sortable themselves...?